src/swarm/neo/client/RequestSet.d

```diff
@@ -351,18 +315,17 @@ public final class RequestSet: IRequestSet
                     can be packed by swarm.neo.util.StructPacker
                 handler = request handler
                 finished_notifier = called when the last hander has finished
-                context = request context
-                working = initial working data to serialize into the request
-                    connections
```

Comment:
The changes from
```
context = request context
```
to
```
context = request context, must be a Contiguous-serialisable
    struct
```
are wrong. See 9c87a4682854d52c8569993073343e9127569f03.

Reply:
Oh, yes!